addons/web_editor/static/src/js/wysiwyg/widgets/alt_dialog.js

```diff
@@ -1,49 +1,72 @@
-/** @odoo-module alias=wysiwyg.widgets.AltDialog **/
+/** @odoo-module **/
 
-import core from "web.core";
-import Dialog from "wysiwyg.widgets.Dialog";
 
-var _t = core._t;
+import { Component, xml, useRef } from "@odoo/owl";
+import { Dialog } from "@web/core/dialog/dialog";
 
-/**
- * Let users change the alt & title of a media.
- */
-var AltDialog = Dialog.extend({
-    template: 'wysiwyg.widgets.alt',
-    /**
-     * @constructor
-     */
-    init: function (parent, options, media) {
-        options = options || {};
-        this._super(parent, Object.assign({}, {
-            title: _t("Change media description and tooltip")
-        }, options));
+export class AltDialog extends Component {
+    static components = { Dialog };
+    static props = {
+        confirm: Function,
+        close: Function,
+        alt: String,
+        tag_title: String,
+    };
+    static template = xml`
+        <Dialog size="'xl'" title="'Change media description and tooltip'">
+            <form action="#">
+                <div class="mb-3 row">
+                    <label class="col-md-3 col-form-label" for="alt"
+                        title="'Alt tag' specifies an alternate text for an image, if the image cannot be displayed (slow connection, missing image, screen reader ...).">
```

Comment:
Are such strings translated when they are not defined via an XML file? I really doubt it, I don't see how to extractor would magically work without custom code in this case :grimacing: